architecture/service-container.md

```diff
@@ -254,13 +254,13 @@ If you need to utilize a container outside the normal flow of Masonite like insi
 This would look something like:
 
 ```python
-from wsgi import container
+from masonite.container import Container
```

Comment:
is this even right? shouldnt it be 

```python
from wsgi import application
from masonite import Queue
...
def handle(self):
        queue = application.make(Queue)
        queue.push(..)
```

Reply:
Honestly I don't know what is the correct way to do it. 
I tried to follow the documentation and I couldn't so if you think that's the correct way I believe you.